chrome.manifest

```diff
@@ -1,5 +1,5 @@
-content   personaswitcher		    chrome/content/
-locale    personaswitcher	en-US	chrome/locale/en-US/
+content   personaswitcher		    	chrome/content/
+locale    personaswitcher	en-US		chrome/locale/en-US/

```

Comment:
Yea, sadly it's in a few places. I can push up a commit just for this if we want, or wait until we go through and clean it up to match whatever we decide the conventions will be.